### PR TITLE
filtering changes for SCIM

### PR DESCRIPTION
**Pull request by mystiberry-okta**
@bng-okta Can you please review?

### CODE REVIEW

File: _source/_standards/SCIM/index.md [hunk 1]
```diff
@@ -609,12 +599,8 @@ your application:
...
 ~~~
 
-Note: The sample application included in this project does not yet
-demonstrate how to implement storing and filtering by
-`externalId`. However, Okta strongly recommends that your SCIM
-implementation supports storing and filtering by `externalId`. For
-details on supporting `externalId`, see
-[section 3.1](https://tools.ietf.org/html/rfc7643#section-3.1) of [RFC 7643](https://tools.ietf.org/html/rfc7643). Quoted below:
+For details about supporting `externalId`, see
+[section 3.1](https://tools.ietf.org/html/rfc7643#section-3.1) of [RFC 7643](https://tools.ietf.org/html/rfc7643):
```
Comment (bng-okta) [inline _source/_standards/SCIM/index.md]: @tgu-okta Re-reading this again and how confusing the term `externalId` can get to an ISV, maybe a clarifying sentence that says in the following excerpt, "provisioning client" in this case is Okta and the service provider is you.

Comment (mystiberry-okta) [inline _source/_standards/SCIM/index.md]: Great idea! Will add.

Comment (tgu-okta) [inline _source/_standards/SCIM/index.md]: @bng-okta - agreed.
@mystiberry-okta - please append the following after "RFC 7643":
"Note that in the following excerpt, "provisioning client" refers to Okta and the service provider refers to you, the 3rd-party that Okta is making calls to."

Comment (mystiberry-okta) [inline _source/_standards/SCIM/index.md]: Done.

**Pull request merged**